### PR TITLE
OPSEXP-1853: use newer alfresco-search-service chart

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1853

### CODE REVIEW

**Review by gionn — APPROVED** [review]

**Comment by alxgomz** [comment]
Tested against the pipeline here (assuming the AAE tests failure are unrelated as everything seems to deploy OK):
https://github.com/Alfresco/terraform-alfresco-pipeline/actions/runs/4862124863

File: helm/alfresco-content-services/values.yaml [hunk 1]
```diff
@@ -918,6 +913,8 @@ alfresco-sync-service:
   # Check [Alfresco Sync Service
   # Documentation](https://github.com/Alfresco/alfresco-helm-charts/tree/main/charts/alfresco-sync-service)
   enabled: true
+  image:
+    tag: 4.0.0-M7
```
Comment (gionn) [inline helm/alfresco-content-services/values.yaml:917]: I have the feeling we need a new epic (?) for moving/splitting the current updatecli to the new helm charts repo (one updatecli config for each chart?) and avoid having any image tag in the parent chart (which of course can be overridden here by users)

File: docker-compose/7.0.N-docker-compose.yml [hunk 2]
```diff
@@ -163,7 +163,7 @@ services:
       - alfresco
       - share
   sync-service:
-    image: quay.io/alfresco/service-sync:3.9.0
+    image: quay.io/alfresco/service-sync:3.7.0
```
Comment (gionn) [inline docker-compose/7.0.N-docker-compose.yml:166]: it would be great to have a separate PR for reverting master

File: helm/alfresco-content-services/templates/config-repository.yaml [hunk 3]
```diff
@@ -30,17 +30,11 @@ data:
...
-      -Dsolr.host={{ template "alfresco-search.host" . }}
-      -Dsolr.port={{ template "alfresco-search.port" . }}
-      -Dsolr.secureComms={{ .Values.global.tracking.auth | default "secret" }}
-      {{- else if index .Values "alfresco-search-enterprise" "enabled" }}
-      {{- include "repo.elasticsearch.config" . | nindent 6 }}
+      {{- $subsys := include "repository.indexSubsystem" . }}
+      {{- if ne $subsys "none" }}
+      {{- include "repository.indexConfig" . | indent 6 }}
       {{- end }}
+      -Dindex.subsystem.name={{ $subsys }}
```
Comment (gionn) [inline helm/alfresco-content-services/templates/config-repository.yaml:37]: I was doing a similar refactor in my PR and it would require a manual merge, better avoid any cleanup outside the current chart being migrated until we finished with migrating charts

Comment (alxgomz) [reply helm/alfresco-content-services/templates/config-repository.yaml:37]: I'm afraid I need this one as explained this morning

File: helm/alfresco-content-services/7.3.N_values.yaml [hunk 4]
```diff
@@ -76,6 +77,5 @@ postgresql-syncservice:
   image:
     tag: 14.4.0
 alfresco-sync-service:
-  syncservice:
-    image:
-      tag: 3.9.0
+  image:
+    tag: 3.8.0
```
Comment (gionn) [inline helm/alfresco-content-services/7.3.N_values.yaml:81]: check this

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

File: helm/alfresco-content-services/7.3.N_values.yaml (hunk 4 above)
Comment (alxgomz) [reply helm/alfresco-content-services/7.3.N_values.yaml:81]: github glitch (probably due to some reverted commit that's been merged in master via another PR in the meantime)
